### DISCUSSION TITLE
NFT价值的来源探索

### DISCUSSION THREAD

**Discussion by 0x1be20** [rebase-network/hello-world#65] [Ideas]
目前NFT的用途主要在数字艺术品收藏以及gamefi里面的，NFT的价值有待于挖掘。

我有一个想法，用NFT作为集资工具。比如我有一个量化策略，发行100个NFT，这销售的收入作为量化策略的本金，策略的收益每隔一段时间发送到NFT持有者手中。这样NFT就有了价值，存在被买卖的价值。

**Comment by luckyyang** [comment]
用 erc20 不是更好吗，流动性更好

**Reply by 0x1be20** [reply]
> 用 erc20 不是更好吗，流动性更好

虽然erc20和nft在一些场景下是作用一样的，但是如果采用erc20的话，收益计算以及发放要麻烦。

如果流动性指的是能够通过uniswap方便购买的话，erc20是方便。但是这样存在一个问题，就是流动池中的代币享受的收益是没法发送到用户手中。如果用户是为了享受策略收益而持有代币的话，那么也不会加入流动池中。这样想的话，其实流动性也并不会很好。

nft的话就没有以上的问题，所以我认为nft在这种场景下比较合适。

**Reply by luckyyang** [reply]
有道理。erc20 其实也可以做，就是麻烦点，要做好计算。我想流动性方面，还有一点是 erc20 是可以拆分的，而 nft 无法拆分，可以拆分的话我可以随时出售一部分权益出去，nft 是不行的

**Reply by 0x1be20** [reply]
NFT还有一个特点，就是**确权**，社区通过NFT来确定拥有者对于社区具有贡献。比如基金通过NFT来确定你对于基金的初始资金有贡献，所以未来能够享有分红的权利。再比如社区的NFT，可以用来证明对于社区的发展有贡献，所以未来社区可以通过NFT来给贡献者提供有价值的服务或者东西。

这种确权的概念，erc20是比较薄弱的。

如果要把权利拆分的话，可以将NFT token化，是NFT化之后的下一个步骤。

**Reply by luckyyang** [reply]
了解了。那如果要确权，那是不是不可转移的 NFT 更好呢
